appyters/DrugShot/DrugShot.ipynb

```diff
@@ -472,11 +518,11 @@
    "outputs": [],
    "source": [
     "%%appyter code_exec\n",
-    "# Calculate average co-mentions for each drug in the prediction matrix with the associated drug set\n",
-    "df_cooccurrence = df_cooccurrence.loc[df_cooccurrence.index.isin(associated_compounds)]\n",
-    "print('%i small molecules matched in the literature co-mentions matrix from the associated drug set' % len(df_cooccurrence.index))\n",
-    "df_cooccurrence.loc['Average Co-mentions'] = df_cooccurrence[df_cooccurrence.columns].mean()\n",
-    "df_cooccurrence.sort_values(by = ['Average Co-mentions'], axis = 1, ascending = False, inplace = True)"
+    "# Calculate average similarity for each drug in the prediction matrix with the associated drug set\n",
+    "mat = mat.loc[mat.index.str.upper().isin(associated_compounds)]\n",
+    "print(f'{len(mat.index)} small molecules from the associated drug set matched in the {{similarity_matrix.raw_value}} matrix')\n",
```

Comment:
Please avoid using `.raw_value` in code cells, and prefer `{{ field }}` (which inserts repr) over `{{ field.value }}`